### PR TITLE
Improve comment on need for -Wno-redundant-constraints

### PR DESCRIPTION
**Pull request by erikd**

### CODE REVIEW

File: cardano-crypto-class/src/Cardano/Crypto/KES/Simple.hs [hunk 1]
```diff
@@ -12,7 +12,9 @@
 {-# LANGUAGE NoStarIsType #-}
 {-# LANGUAGE MultiParamTypeClasses #-}
 
--- Needed for ghc-9.6.
+-- Needed for ghc-9.6 to avoid a redunant constraint warning on the
+-- `KESSignAlgorithm m (SimpleKES d t)` instance. Removing the constraint leaves another type
+-- error which is rather opaque.
 {-# OPTIONS_GHC -Wno-redundant-constraints #-}
```
Comment (angerman) [inline cardano-crypto-class/src/Cardano/Crypto/KES/Simple.hs:18]: @lehins maybe something you can take a look at when you are having a free minute?

File: .github/workflows/build-blst.bash [hunk 2]
```diff
@@ -10,13 +10,13 @@ git reset --hard $BLST_REF
...
 mkdir -p ${PREFIX}/include/blst
-cp bindings/{blst.h,blst_aux.h} ${PREFIX}/include/blst/ 
+cp bindings/{blst.h,blst_aux.h} ${PREFIX}/include/blst/
 cp -f libblst.{a,dll,so,dylib} ${PREFIX}/lib/
 
 cat <<EOF > ${PREFIX}/lib/pkgconfig/libblst.pc
 prefix=${PREFIX}
 exec_prefix=\${prefix}
-libdir=\${prefix}
+libdir=\${prefix}/lib
```
Comment (angerman) [inline .github/workflows/build-blst.bash:19]: oops! Thanks for spotting that!

**Review by angerman — APPROVED** [review]
LGTM!

File: cardano-crypto-class/src/Cardano/Crypto/KES/Simple.hs (hunk 1 above)
Comment (lehins) [reply cardano-crypto-class/src/Cardano/Crypto/KES/Simple.hs:18]: Sounds good, I'll take a look at it.

I've seen similar behavior in one of my libraries with ghc-9.4, which complains about redundant constraint, but removing the constraint breaks ghc-8

So, we might have to add CPP for the GHC flag in order to accommodate older ghc versions

**Pull request merged**